tpm2/policy_test.go

```diff
@@ -62,25 +58,25 @@ func TestIncrementPcrPolicyCounter(t *testing.T) {
 		undefineNVSpace(t, tpm, index, tpm.OwnerHandleContext())
 	}()
 
-	initialCount, err := ReadPcrPolicyCounter(tpm.TPMContext, CurrentMetadataVersion, policyCounterPub, nil, tpm.HmacSession())
+	h, err := NewPcrPolicyCounterHandleV1(policyCounterPub, keyPublic)
 	if err != nil {
-		t.Errorf("ReadPcrPolicyCounter failed: %v", err)
+		t.Fatalf("NewPcrPolicyCounterHandleV1 failed: %v", err)
 	}
 
-	if err := IncrementPcrPolicyCounter(tpm.TPMContext, CurrentMetadataVersion, policyCounterPub, nil, key, keyPublic, tpm.HmacSession()); err != nil {
-		t.Fatalf("IncrementPcrPolicyCounter failed: %v", err)
+	if err := h.Set(tpm.TPMContext, initialCount+1, key, tpm.HmacSession()); err != nil {
```

Comment:
is there a check that Set supports a delta larger than 1, we don't use that, but the code is supposed to support and with some complexity so it would be good to test for that?

Reply:
I've added a test for that now - the delta will be larger than one if you call UpdatePCRProtectionPolicy more than once before calling RevokeOldPCRProtectionPolicies.